draft-ietf-tls-dtls-connection-id.md

```diff
@@ -400,7 +400,8 @@ communication).  Without multi-homing or mobility, the use of the CID
 is not different to the use of the 5-tuple.
 
 With multi-homing, an adversary is able to correlate the communication
-interaction over the two paths, which adds further privacy concerns.
+interaction over the two paths, which adds further privacy concerns. The lack 
+of a CID update mechanism makes this extension unsuitable for mobility scenarios.
```

Comment:
I would prefer, to make the "unsuitable" conditional. Therefore I would propose to add something as "when this possible correlation must be considered" . 

My concern here is, that with cid you can communicate perfectly in mobility scenarios, but you may get tracked. Without the condition someone easily misunderstands the "unsuitable", even if the scenario doesn't require "track protection".